components/raftstore/src/store/snap.rs

```diff
@@ -1118,14 +1118,14 @@ struct SnapManagerCore {
 /// `SnapManagerCore` trace all current processing snapshots.
 pub struct SnapManager {
     core: SnapManagerCore,
-    max_total_size: u64,
+    max_total_size: AtomicU64,
```

Comment:
I guess it's also necessary to hold a `cfg_tracker` here to update `max_write_bytes_per_sec` and then call `self.set_speed_limit` for snapshot generating.